### DISCUSSION TITLE
Every time i update my javascript code i have to do "ctrl + shift + R".

### DISCUSSION THREAD

**Discussion by 7ib0** [community/community#132507] [Programming Help]
### Body

If i dont do "ctrl + shift + R" i dont see the changes, i have asked my friend to test this and he has the same issue. is theres a way to add this in the code that it does it automatically?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by thiunuwan** [comment]
You could use Live Server to automatically refresh your page with changes

 Install Live Server Extension:

#### Open Visual Studio Code.
Go to the Extensions view by clicking on the Extensions icon in the Activity Bar on the side of the window or by pressing Ctrl+Shift+X.
Search for "Live Server" and install the extension by Ritwick Dey.
#### Open Your Project Folder:

Open your project folder in Visual Studio Code.
#### Start Live Server:

Right-click on your index.html file (or any other HTML file you want to serve) in the Explorer pane.
Select "Open with Live Server".
Alternatively, you can click on the "Go Live" button in the bottom right corner of the status bar.

**Reply by 7ib0** [reply]
This is not really what im looking for because im trying to make a public website....

**Comment by x-INFiN1TY-x** [comment]
Hey 7ib0,

The issue you're encountering where changes in your JavaScript code aren't reflected until you perform a hard refresh (Ctrl + Shift + R) is likely due to browser caching. Browsers cache files like JavaScript to improve performance, which can sometimes lead to displaying older versions of files after they've been updated.
